Tests/FirebaseTests/Models/DynamicLinkRequest+Dummy.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+@testable import Firebase
+
+extension DynamicLinkRequest {
+
+    static var dummy: DynamicLinkRequest {
+        return DynamicLinkRequest(dynamicLink: DynamicLinkInfo(link: "http://link"), suffix: Suffix(option: .SHORT))
+    }
```

Comment:
same here, you can inline this and get rid of the braces

Reply:
41ba0cf